src/tests/CMakeLists.txt

```diff
@@ -128,6 +128,21 @@ if(Python3_Interpreter_FOUND)
             output_tests(false, "sirius")
         endif()
         
+        # Specific tests
+        add_test(
+            NAME short-specific
```

Comment:
replace "specific" by "infeasibility" ?

Reply:
"specific" choose over "infeasibility" if we want to add more pytest for other functionnality